proposal.md

```diff
@@ -5,3 +5,13 @@
 ## Description of the Data
 
 ## Research Questions and Usage Scenarios
+
+### Research Questions
+
+Doggodash is designed to help potential dog owners find their favourite dog breeds by answering important questions they have before deciding to own one of them. Many dog breeds have certain consistent and standard attributes such as coat length, energy levels, young children friendly and many more. However, too many attributes can be confusing. 
+
+Therefore, with the option to select few but most important attributes, Doggodash with its interactive dashboard application can help answer the simplest yet most effective question - given certain attribute preferences, which are the best dogs to have.
+
+### Usage scenario
+
+Susane and David are a married couple, both in their thirties with two kids who are 2 years and 5 years old. They recently decided to add a pet dog to their family. They found several pet websites on the internet but with the overload of information, they are unable to decide on the best dog breed to have. Susane has a preference for energetic dogs while David has a preference for dogs with low coat length and low shedding frequency. When Susane and David log on to the "Doggodash application", they will see a number of attributes to choose as filters. The maximum number of attributes they can choose as filters is 4. Once, they have choosen their preferred attributes as "filters", they will now tweak these filters based on a scale of 1 to 5 (low to high). Once done, the application displays  the top 5 dog breeds as per the choosen and tweaked attributes. With these  top five dog breeds, she now enquires about their prices, age, availability and other health related aspects of these breeds from the nearest pet shop and plans a visit to finalize one.
```

Comment:
Please change the following:
"internet" to "Internet"
"Doggodash" to "DoggoDash" (I suggest that the common name to be capitalized in this style while the technical name of the repo remain `doggodash` in all small letters.)